src/dodal/devices/ophyd_async_xspress3_mini/xspress3_mini_oa.py

```diff
@@ -0,0 +1,109 @@
+from enum import Enum
+
+from ophyd_async.core import (
+    AsyncStatus,
+    Device,
+    DeviceVector,
+    set_and_wait_for_value,
+    wait_for_value,
+)
+from ophyd_async.epics.signal.signal import (
+    epics_signal_r,
+    epics_signal_rw,
+    epics_signal_rw_rbv,
+)
+
+from dodal.devices.ophyd_async_xspress3_mini.xspress3_mini_channel_oa import (
+    Xspress3MiniChannel,
+)
+
+from dodal.log import LOGGER
+
+
+class AttenuationOptimisationFailedException(Exception):
+    pass
+
+
+class TriggerMode(str, Enum):
+    SOFTWARE = "Software"
+    HARDWARE = "Hardware"
+    BURST = "Burst"
+    TTL_Veto_Only = "TTL_Veto_Only"
+    IDC = "IDC"
+    SOTWARE_START_STOP = "Software_Start/Stop"
+    TTL_BOTH = "TTL_Both"
+    LVDS_VETO_ONLY = "LVDS_Veto_Only"
+    LVDS_both = "LVDS_Both"
+
+
+class UpdateRBV(str, Enum):
+    DISABLED = "Disabled"
+    ENABLED = "Enabled"
+
+
+class EraseState(str, Enum):
+    DONE = 0
+    ERASE = 1
+
+
+class AcquireState(str, Enum):
+    DONE = 0
+    ACQUIRE = 1
+
+
+class DetectorState(str, Enum):
+    ACQUIRE = "Acquire"
```

Comment:
Why capitalize the first letter?

Reply:
it is to match the c enum so I can either match "Acquire "or use the location in this case 0 to set them. Which is also the reason I have not change done = 0 above to string yet... I need to find out what it is.